### ISSUE TITLE
[BUG] Bashtop crashes when the internet is turned off / not connected

### ISSUE THREAD

**Issue by nikhil-jhaveri** [aristocratos/bashtop#115]
Error log from the log file located at: $HOME/.config/bashtop/error.log
```
New instance of bashtop version: 0.8.29 Pid: 14594
df: /run/user/1000/doc: Operation not permitted
08:56:59 PM ERROR: On line 2011
RTNETLINK answers: Network is unreachable
08:56:59 PM ERROR: On line 315
08:56:59 PM ERROR: On line 315
/home/someuser/bashtop-0.8.29/bashtop: line 1485: *100/max: syntax error: operand expected (error token is "*100/max")
```
My Platform:
OS: Manjaro 20.0.1
WM: KDE
Terminal: Konsole
bash version: 5.0.16(1)-release

PS: I think this has been happening since the max and min network stat was added to the network widget. **This problem does not occur in Bashtop v0.8.27**

Let me know if you need more info.

**Comment by aristocratos** [comment]
Will put up a fix later, that error checks the network and periodically checks for new network if disconnect.

**Comment by aristocratos** [comment]
@nikhil-jhaveri Fixed in v0.8.30
